src/aggregator/client/writer.go

```diff
@@ -464,5 +464,5 @@ func newRefCountedWriter(instance placement.Instance, opts Options) *refCountedW
 }
 
 func (rcWriter *refCountedWriter) Close() {
-	rcWriter.instanceWriter.Close() // nolint: errcheck
+	go rcWriter.instanceWriter.Close() // nolint: errcheck
```

Comment:
Can you add a `NOTE`/`NB` here to why this is asynchronous?

Reply:
Sure, sounds good